tests/dummy_tests.rs

```diff
@@ -174,6 +174,31 @@ mod already_connected {
 
     use tokio_util::codec::{Framed, LinesCodec};
 
+    #[tokio::test]
+    async fn back_to_back_shutdown_attempts() {
+        use stubborn_io::StubbornTcpStream;
+        use tokio::io::AsyncWriteExt;
+
+        const ADDR: &str = "127.0.0.1:3989";
```

Comment:
This isn't strictly necessary, but to make the test more flexible (what if port 3989 is already taken), ideally you could [bind](https://doc.rust-lang.org/std/net/struct.TcpListener.html#method.bind) to port 0, which will assign a random available port and then you can query the connection to see what the port is.

Youd have to do something like use a tokio oneshot sender and send the port value from inside your tokio::spawn block in order to have it communicated to when you for the StubbornTcpStream connection.

Reply:
Yep that sounds good, I'll address this and the above issue when resolving the rest of the changes

Reply:
Fixed in 1a35332